app/src/main/java/mod/jbk/build/BuiltInLibraries.java

```diff
@@ -114,7 +114,6 @@ public class BuiltInLibraries {
     public static String JETBRAINS_ANNOTATIONS = "annotations-13.0";
     public static String KOTLIN_STDLIB = "kotlin-stdlib-1.6.21";
     public static String KOTLIN_STDLIB_JDK7 = "kotlin-stdlib-jdk7-1.6.21";
```

Comment:
You should remove too

Reply:
its already exist in sketchware but ok why not i will remove it

Reply:
done ✅️